presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -1321,4 +1321,14 @@ public void testInverseNormalCdf()
         assertInvalidFunction("inverse_normal_cdf(4, 48, 1)", "p must be 0 > p > 1");
         assertInvalidFunction("inverse_normal_cdf(4, 0, 0.4)", "sd must > 0");
     }
+
+    @Test
+    public void testNormalCdf()
+            throws Exception
+    {
+        assertFunction("normal_cdf(0, 1, 1.96)", DOUBLE, 0.9750021048517796);
```

Comment:
It would be useful to add tests for nan/infinity.

Reply:
@nezihyigitbasi I added a few tests for if v = +/- infinity, but not sure how presto handles nans / infinity. What's the convention here? nans produce nans? nans produce errors ... ?

Reply:
Many (most?) Presto functions don't fail with nan input, instead they return a nan value. The return value depends on the function though. In the general case an arbitrary function can also return a non-nan value for a nan input (e.g., java.lang.Double::doubleToLongBits()) .

Reply:
@nezihyigitbasi I added several tests relating to nans and infinity, PTAL